Assets/Scripts/Models/Tile.cs

```diff
@@ -112,7 +112,7 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovmentModifier { get; set; }
+    public float MovementModifier { get; set; }
```

Comment:
That was the idea, to be able to change the movmentcost of a tile on a tile by tile basis